### DISCUSSION TITLE
WalletConnect Dashboard migration

### DISCUSSION THREAD

**Discussion by bunbuarka** [WalletConnect/walletconnect-monorepo#2799] [Developer Support] [answered]
Hi!

Any methods on moving walletconnect dashboard account from one to another? 
or is there any method to change the ownership of dashboard account safely?

**Comment by finessevanes** [comment]
@bunbuarka by WalletConnect dashboard, do you mean the Cloud project settings?

**Comment by JeroldG** [comment, marked as answer]
Hi we can transfer for you, if you could send  an email to cloud@walletconnect.com from the email that is linked to the account and mention the new email that you want to transfer to. The new email needs to be signed up so I would recommend that we signup here. https://cloud.walletconnect.com/sign-up
